prow/cmd/deck/static/pr-script.js

```diff
@@ -786,6 +838,30 @@ function createMergeStatus(prLabels, queries) {
     return statusContainer;
 }
 
+/**
+ * Creates the merge conflict status.
+ * @param prMergeability
+ * @return {Element}
+ */
+function createMergeConflictStatus(prMergeability) {
+    const statusContainer = document.createElement("DIV");
+    statusContainer.classList.add("status-container");
```

Comment:
hmm, we probably should tweak this, there really shouldn't be a line below the last status

Reply:
IE the style of this status is different from the tests and labels because it's not an expandable object.